### DISCUSSION TITLE
Interpretable chemical information from Avalon fingerprints

### DISCUSSION THREAD

**Discussion by canozkan42** [rdkit/rdkit#7520] [Q&A]
Hi all, 
Is this possible? I managed to do this for Morgan, Rdkit and atompair fingerprints with the help of rdkit.Chem.rdFingerprintGenerator additionalOutput, however generator method doesn't support Avalon yet and I can't find the option to do this through rdkit.Avalon.pyAvalonTools. I am doing this to check whether there is hash level bit collision, which I saw was very prominent in rdkit path based fingerprints.

**Comment by manajit-das** [comment]
I think currently it is not possible to get the chemical information from Avalon fingerprints as mentioned by @greglandrum
here  https://github.com/rdkit/rdkit/discussions/5495#discussioncomment-3510993
